torch/_dynamo/trace_rules.py

```diff
@@ -84,7 +81,2617 @@
 }
 
 
-torch_name_rule_map = {**manual_torch_name_rule_map, **auto_torch_name_rule_map}
+# In graph functions (including constant folding) that are C bindings
+torch_C_binding_in_graph_functions = {
```

Comment:
There are a few functions in ```torch_C_binding_in_graph_functions``` and ```torch_non_C_binding_in_graph_functions``` which shouldn't be here, I'll do an audit pass and remove inaccurate ones with my best effort in next PR. After that, we can still gradually correct and improve this list to make it reach ~100% accurate.

Reply:
The variable name here implies everything here will map to `TorchInGraphFunctionVariable`.  Will that always be true?  Or do we want people to change this in the future (for example to turn something into a graph break).
1) If it will change we should use a more future-proof variable name.
2) If it won't change, we should make this a list and do:
`{k: TorchInGraphFunctionVariable for k in ...}`

Reply:
Good question! If people want to add/remove item in an ad-hoc way, they should go to the ```manual_torch_name_rule_map```. E.g, they can add a item ```torch.x.y : SkipFilesVariable``` to indicate a graph break. I updated with the second option you suggested.